cmd/proxy-server/main.go

```diff
@@ -24,6 +24,12 @@ var flags []cli.Flag = []cli.Flag{
 		Value:   "127.0.0.1:8080",
 		Usage:   "address to listen on",
 	},
+	&cli.StringFlag{
+		Name:    "listen-addr-healthcheck",
+		EnvVars: []string{"LISTEN_ADDR_HEALTHCHECK"},
+		Value:   "",
```

Comment:
should we also provide a default value for the health check similar to the other flags?

Reply:
i think it's better to make opt-in for now, instead of starting it by default.